### PR TITLE
add test for min_max_schema_for_fields

### PR DESCRIPTION
**Pull request by marijncv**
Signed-off-by: Marijn Valk <marijncv@hotmail.com>

# Description
Adds tests for `min_max_schema_for_fields`

# Related Issue(s)

# Documentation

### CODE REVIEW

File: rust/src/delta_arrow.rs [hunk 1]
```diff
@@ -732,4 +732,63 @@ mod tests {
...
+        let f = ArrowField::new(
+            "struct",
+            ArrowDataType::Struct(vec![ArrowField::new("simple", ArrowDataType::Int32, true)]),
+            true,
+        );
+        let expected = vec![ArrowField::new(
+            "struct",
+            ArrowDataType::Struct(vec![ArrowField::new("simple", ArrowDataType::Int32, true)]),
+            true,
+        )];
```
Comment (wjones127) [inline rust/src/delta_arrow.rs:757]: We don't have to repeat this, and it's probably clearer they are meant to be the same field if we just clone `f`. Same applies for test above.
```suggestion
        let f = ArrowField::new(
            "struct",
            ArrowDataType::Struct(vec![ArrowField::new("simple", ArrowDataType::Int32, true)]),
            true,
        );
        let expected = vec![f.clone()];
```

**Review by wjones127 — CHANGES_REQUESTED** [review]
Since `max_min_schema_for_fields` is meant to be used multiple times on the same destination, perhaps it might be better to combine the tests into one where you call the function on each of the fields, and then assert the final `dest` result contains the right fields.

**Comment by marijncv** [comment]
Thanks for the pointers @wjones127!

**Review by wjones127 — APPROVED** [review]
Thanks @marijncv!

**Pull request merged**